Lib/fontTools/t1Lib/__init__.py

```diff
@@ -255,13 +251,10 @@ def writePFB(path, data):
 			f.write(longToString(len(chunk)))
 			f.write(chunk)
 		f.write(bytechr(128) + bytechr(3))
-	finally:
-		f.close()
```

Comment:
the `try: finally:` was doing exactly the same thing, but if you prefer using `with`, I'm fine with it.